ansible/setup_vs_chassis.sh

```
@@ -0,0 +1,262 @@
+# Function to set up midplane address and wait for supervisor
+setup_midplane_and_wait_for_supervisor() {
+    local slot_num="$1"
+    if [ "$is_linecard" = true ]; then
+        echo "Setting up midplane address for linecard: $slot_num"
+        sudo ip -d addr show eth1
+        sudo ip link set up dev eth1
+        sudo ip addr add 10.6.$slot_num.100/16 dev eth1
+
+        echo "Waiting for supervisor to become reachable"
+        counter=0
+        while [! ping -c 1 10.6.0.100 &> /dev/null && counter <= 300]; do
```

Shellcheck (0.10.0)

[error] You need a space after the [ and before the ].

(SC1035)